.gitlab/package_build/deb.yml

```diff
@@ -34,7 +34,10 @@
     - $S3_CP_CMD $S3_ARTIFACTS_URI/conntrack.c.${PACKAGE_ARCH} /tmp/system-probe/conntrack.c
     - $S3_CP_CMD $S3_ARTIFACTS_URI/oom-kill.c.${PACKAGE_ARCH} /tmp/system-probe/oom-kill.c
     - $S3_CP_CMD $S3_ARTIFACTS_URI/tcp-queue-length.c.${PACKAGE_ARCH} /tmp/system-probe/tcp-queue-length.c
-    - chmod 755 /tmp/system-probe/system-probe
+    - $S3_CP_CMD $S3_PERMANENT_ARTIFACTS_URI/clang-11.0.1.${PACKAGE_ARCH} /tmp/system-probe/clang-bpf
+    - $S3_CP_CMD $S3_PERMANENT_ARTIFACTS_URI/llc-11.0.1.${PACKAGE_ARCH} /tmp/system-probe/llc-bpf
+    - chmod 0644 /tmp/system-probe/*.o
```

Comment:
I see `deb` has this, but the other package builds don't? Should they be in sync?